package.json

```diff
@@ -22,18 +22,42 @@
     "/data/**/*.*(js|d.ts)",
     "/fixtures/**/*.*(js|d.ts)",
     "/har/**/*.*(js|d.ts)",
+    "/storybook/**/*.*(js|d.ts)",
     "/utils/**/*.*(js|d.ts)",
     "!/**/__tests__"
   ],
-  "exports": [
-    "./actions/**/*.*(js|d.ts)",
-    "./auth/**/*.*(js|d.ts)",
-    "./config/**/*.*(js|d.ts)",
-    "./data/**/*.*(js|d.ts)",
-    "./fixtures/**/*.*(js|d.ts)",
-    "./har/**/*.*(js|d.ts)",
-    "./utils/**/*.*(js|d.ts)"
-  ],
+  "exports": {
+    "./actions": {
+      "types": "./actions/index.d.ts",
+      "import": "./actions/index.js",
+      "require": "./actions/index.js"
```

Comment:
Maybe we can just use `default` here, instead of `import`+`require` with the same target?

> "default" - the generic fallback that always matches. Can be a CommonJS or ES module file. This condition should always come last.